cmd/gobrew/main.go

```diff
@@ -142,6 +142,12 @@ func Find(slice []string, val string) (int, bool) {
 }
 
 func usage() string {
+	usageMsg :=
```

Comment:
I am not sure if we need `main_unix.go` or `main_windows.go` just for those const.
It displays unwanted warning on the editor

![Screenshot 2023-12-12 at 11 20 18](https://github.com/kevincobain2000/gobrew/assets/629055/fe566eff-4f2d-4575-b8c8-630fafd370d8)

Also makes development process complicated.

```
╰─$ go run cmd/gobrew/main.go -h
# command-line-arguments
cmd/gobrew/main.go:178:5: undefined: usageMs
```